test.js

```diff
@@ -0,0 +1,78 @@
+require( 'sugar' );
+var assert = require( 'assert' );
+var Promise = require( 'bluebird' );
+var parseBig = require( './index' );
+
+var MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER;
+var MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER;
+
+describe( 'Parse Big', function () {
+    it( 'error', function ( done ) {
```

Comment:
This would generate a test output such as:
```
Parse big error:
 AssertionError ....
```

A test name should describe the desired behaviour, just as an example
```js
it('throws an error on unparseable strings', funciton ()...
``` 

This would generate `Parse Big throws an error on unparseable strings` which reads much more easily.
Same comment goes for `parse` test.